### PR TITLE
fix(core/inline-idl-parser): support nullable IDL types

### LINKED ISSUE #3432: Support linking nullable IDL container types
**Is your feature request related to a problem? Please describe.**

I have to write {{PositionErrorCallback}}`?`...

**Describe the solution you'd like**

But would like to write {{PositionErrorCallback?}} and have PositionErrorCallback linked, but shown as nullable.

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #3432

### CODE REVIEW

File: src/core/inlines.js [hunk 1]
```diff
@@ -61,8 +61,8 @@ const l10n = getIntlData(localizationStrings);
 // TODO: Replace (?!`) at the end with (?:<!`) at the start when Firefox + Safari
 // add support.
 const inlineCodeRegExp = /(?:`[^`]+`)(?!`)/; // `code`
-const inlineIdlReference = /(?:{{[^}]+}})/; // {{ WebIDLThing }}
-const inlineVariable = /\B\|\w[\w\s]*(?:\s*:[\w\s&;<>]+)?\|\B/; // |var : Type|
+const inlineIdlReference = /(?:{{[^}]+\?*}})/; // {{ WebIDLThing? }}
```
Comment (sidvishnoi) [inline src/core/inlines.js:64]:
```suggestion
const inlineIdlReference = /(?:{{[^}]+\?*}})/; // {{ WebIDLThing }}, {{ WebIDLThing? }}
```

File: src/core/inlines.js [hunk 2]
```diff
@@ -61,8 +61,8 @@ const l10n = getIntlData(localizationStrings);
...
+const inlineVariable = /\B\|\w[\w\s]*(?:\s*:[\w\s&;<>]+\??)?\|\B/; // |var : Type|
```
Comment (sidvishnoi) [inline src/core/inlines.js:65]: This change appears to be leaked from https://github.com/w3c/respec/pull/3627

**Review by sidvishnoi — APPROVED** [review]
LGTM. Few nits.

File: src/core/inlines.js (hunk 2 above)
Comment (marcoscaceres) [reply src/core/inlines.js:65]: oh woopsy. Will merge that first.

**Pull request merged**